charts/seed-monitoring/charts/core/charts/prometheus/values.yaml

```diff
@@ -260,6 +263,56 @@ allowedMetrics:
   - envoy_tcp_downstream_cx_rx_bytes_total
   - envoy_tcp_downstream_cx_total
   - envoy_tcp_downstream_cx_tx_bytes_total
+  reversedVPN:
+  - reversed_vpn_envoy_cluster_external_upstream_rq
```

Comment:
the prefix has to be removed here as well

Reply:
Adapted accordingly with Sebastian's latest commit.